lang/en/docs/cli/unlink.md

```diff
@@ -4,9 +4,11 @@ guide: docs_cli
 layout: guide
 ---
 
+{% include vars.html %}
```

Comment:
I’m pretty sure that this is already included in `default.html`, but should be checked